src/plugins/virtual_scroll/plugin.ts

```diff
@@ -207,8 +207,19 @@ export default function (this: TomSelect) {
 				return;
 			}
 
+			// Save scroll position.
+			const scrollTop = dropdown_content.scrollTop;
+
 			loading_more = true;
 			self.load.call(self, self.lastValue);
+
+			// Restore scroll position after loading more results.
+			const observer = new MutationObserver(() => {
```

Comment:
It would be worth seeing if there is an event that fires after new options are loaded. Might be something like `self.on('load', ...)`

@saifsultanc can you look into this to see if this could be handled with an event callback rather than mutation observer?

Reply:
@veryspry Not sure what I was doing wrong, but didnt manage to get it to work with that